tests/core/failover/task.yaml

```
@@ -130,9 +130,7 @@ execute: |
 
     # and the boot env vars are correctly set
     echo "Waiting for snapd to clean snap_mode"
-    while [ "$("$TESTSTOOLS"/boot-state bootenv show snap_mode)" != "" ]; do
-        sleep 1
-    done
+    retry -n 200 --wait 1 sh -c "test -z "$("$TESTSTOOLS"/boot-state bootenv show snap_mode)""
```

AFAIU the command should be single quoted, otherwise `boot-state bootenv show` is evaluated only once.
```suggestion
    retry -n 200 --wait 1 sh -c 'test -z "$("$TESTSTOOLS"/boot-state bootenv show snap_mode)"'
```
TESTSTOOLS should have been exported, so it should properly resolve inside the subshell.

---

Fixed that, couldn't use ' instead of " because it makes shellcheck unhappy:
retry -n 30 --wait 1 sh -c 'test "$(find "$HOME"/PDF -name "*.pdf" | wc -l)" -ge 1'
^-- SC2148: Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.
                           ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.